src/dxgi/dxgi_options.cpp

```diff
@@ -92,7 +93,7 @@ namespace dxvk {
     this->hideIntelGpu = config.getOption<Tristate>("dxgi.hideIntelGpu", Tristate::Auto) == Tristate::True;
 
     this->enableHDR = config.getOption<bool>("dxgi.enableHDR", env::getEnvVar("DXVK_HDR") == "1");
-    if (this->enableHDR && isHDRDisallowed()) {
+    if (this->enableHDR && isHDRDisallowed() && !nvapiEnabled) {
```

Comment:
Extracting ` env::getEnvVar("DXVK_ENABLE_NVAPI") == "1"` into a function and already checking within `isHDRDisallowed()` might improve readability. Please let me know what you prefer.